src/main/resources/jira-template.ftl

```diff
@@ -1,16 +1,25 @@
-<#assign rowTotalDuration = 0?number>
 <#if getDescription()?has_content>${getDescription()}</#if>
+
 <#if getNarrativeType() == "NARRATIVE_AND_TIME_ROW_ACTIVITY_DESCRIPTIONS">
  <#list getTimeRows() as timeRow>
-  <#assign rowTotalDuration += timeRow.getDurationSecs()>
   |${timeRow.getActivity()}|${timeRow.getDescription()!"N/A"}|
  </#list>
 </#if>
-<#if getDescription()?has_content || getNarrativeType() == "NARRATIVE_AND_TIME_ROW_ACTIVITY_DESCRIPTIONS">
- ${'\n'}
-Total worked time: ${rowTotalDuration?string.@duration}
+
+Total worked time: ${getTotalDuration(getTimeRows())?string.@duration}
 Total chargeable time: ${getTotalDurationSecs()?string.@duration}
```

Comment:
Chargeable time doesn't make sense in the context of Jira. For Jira we just want to use the `workedTime` that we calculated in the `JiraConnector` and that we saved into the database.